source/common/sequencer_impl.cc

```diff
@@ -14,12 +14,12 @@ SequencerImpl::SequencerImpl(const PlatformUtil& platform_util,
                              Envoy::MonotonicTime start_time, RateLimiterPtr&& rate_limiter,
                              SequencerTarget target, StatisticPtr&& latency_statistic,
                              StatisticPtr&& blocked_statistic, std::chrono::microseconds duration,
-                             std::chrono::microseconds grace_timeout)
+                             std::chrono::microseconds grace_timeout, bool use_spin_loop)
```

Comment:
I think you'd want this to be the `enum class` type I suggested above, rather than a bool.